serverMiddleware/server-pulsar-api-proxy.js

```diff
@@ -1,6 +1,12 @@
 const express = require('express')
 const request = require('request')
 const connections = require('./connections')
+const fs = require('fs')
+
+// Read client cert options
+ca = process.env.CLIENT_CA_PATH ? fs.readFileSync(process.env.CLIENT_CA_PATH) : null
```

Comment:
Maybe we should prefix these with `PE_` since that is used for the other env vars.